share/tasks.py

```diff
@@ -44,7 +44,12 @@ def transform(self, raw_id):
         graph = transformer.transform(raw)
 
         if not graph or not graph['@graph']:
-            logger.warning('Graph was empty for %s, skipping...', raw)
+            if not raw.normalizeddata_set.exists():
+                logger.warning('Graph was empty for %s, setting no_change to True', raw)
+                RawDatum.objects.filter(id=raw_id).update(no_change=True)
```

Comment:
Oh right, that makes sense. I was hung up on "no change" rather than thinking about the janitor. 

Would it make more sense for the janitor to look for raws without IngestJobs (once those are a thing)?